components/browse/BrowseContentCard.vue

```diff
@@ -21,6 +22,10 @@
       fields: {
         type: Object,
         default: () => {}
+      },
+      cardType: {
+        type: Object,
```

Comment:
Let's make this a string and have the parent component be responsible for drilling down into the object to extract it.